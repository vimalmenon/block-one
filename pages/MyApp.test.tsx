import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { Router } from 'next/router'
import MyApp from './_app'

jest.mock('../styles/globals.css', () => ({}))
jest.mock('tailwindcss/tailwind.css', () => ({}))
const Components = () => {
  return <div></div>
}
describe('MyApp Page', () => {
  test('MyApp: Snapshot testing', () => {
    const pageProps = {
      test: 'vimal',
    }
    const router = {} as Router

    const wrapper = shallow(<MyApp pageProps={pageProps} Component={Components} router={router} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
