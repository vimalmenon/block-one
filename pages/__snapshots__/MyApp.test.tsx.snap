// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyApp Page MyApp: Snapshot testing 1`] = `
<Components
  test="vimal"
/>
`;
